cluster/ci/monitoring/mixins/grafana_dashboards/e2e_template_jobs.jsonnet

```diff
@@ -0,0 +1,112 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local dashboardConfig = {
+  uid: '68542739-4831-43e0-b5a0-5117369c83c8',
+};
+
+dashboard.new(
+  'e2e template jobs dashboard',
+  time_from='now-12h',
+  schemaVersion=18,
+)
+
+.addPanel(
```

Comment:
Can we define a function for these?

Reply:
Nope, but I did create a variable with the legend config.